kuma/wiki/models.py

```diff
@@ -1735,7 +1735,10 @@ def get_tidied_content(self):
                 # already and don't need to schedule it the next time
                 # we use 3 days as a limit to try it again
                 memcache.set(tidying_scheduled_cache_key, 1, 60 * 60 * 24 * 3)
-            tidied_content, errors = tidy_content(self.content)
+            if allow_blank:
+                tidied_content = ''
```

Comment:
I'm worried that an empty string is a valid tidied content.  I'd rather return something non-string, like ``None`` (changing param to ``allow_null`` or ``async``), to specify that no content was returned.